test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -27,6 +26,8 @@ public class OAuthBearerMiddlewareTests
         {
             var server = CreateServer(options =>
             {
+                options.AutomaticAuthentication = true;
```

Comment:
Well I'm not sure we want to get into "If X is present Y should not be automatic." That's just begging for pain.

If you have both cookie and bearer well the principle is going to have two identities on it, which is what we want anyway right?